src/Sylius/Bundle/CoreBundle/CatalogPromotion/Exception/InvalidCatalogPromotionStateException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\CatalogPromotion\Exception;
+
+final class InvalidCatalogPromotionStateException extends \RuntimeException
```

Comment:
How about putting it in the `Promotion` component? 🤔

Reply:
Done ✋